### PR TITLE
Getting started guides overhaul.

### PR DESCRIPTION
**Pull request by jeffmendoza**
Reference:
https://groups.google.com/forum/#!topic/kubernetes-sig-docs/0WX9pq3Un1E

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2015)

### CODE REVIEW

File: _data/guides.yml [hunk 1]
```diff
@@ -144,13 +137,14 @@ toc:
       - docs/getting-started-guides/fedora/flannel_multi_node_cluster.md
       - docs/getting-started-guides/centos/centos_manual_config.md
       - docs/getting-started-guides/coreos/index.md
-      - /docs/getting-started-guides/ubuntu/
+      - docs/getting-started-guides/ubuntu.md
```
Comment (jaredbhatti) [inline _data/guides.yml]: Broken link

File: _data/guides.yml [hunk 2]
```diff
@@ -114,18 +105,20 @@ toc:
   - title: Running Kubernetes on Custom Solutions
     section:
     - docs/getting-started-guides/scratch.md
+    - docs/getting-started-guides/kubeadm.md
     - title: Custom Cloud Solutions
       section:
       - docs/getting-started-guides/coreos/index.md
-      - /docs/getting-started-guides/juju/
+      - docs/getting-started-guides/juju.md
```
Comment (jaredbhatti) [inline _data/guides.yml]: Broken link

File: _data/guides.yml [hunk 3]
```diff
@@ -114,18 +105,20 @@ toc:
...
       - docs/getting-started-guides/rackspace.md
+      - docs/getting-started-guides/kops.md
     - title: On-Premise VMs
       section:
       - docs/getting-started-guides/coreos/index.md
       - docs/getting-started-guides/cloudstack.md
       - docs/getting-started-guides/vsphere.md
       - docs/getting-started-guides/photon-controller.md
-      - /docs/getting-started-guides/juju/
+      - docs/getting-started-guides/juju.md
```
Comment (jaredbhatti) [inline _data/guides.yml]: page not found

**Comment by jaredbhatti** [comment]
A lot of these links have moved/changed.  Please verify them.

**Comment by jaredbhatti** [comment]
@jeffmendoza  Ping?

**Comment by jeffmendoza** [comment]
Thanks for the ping, I've been preoccupied,  I'll get this updated soon.

**Comment by jeffmendoza** [comment]
@jaredbhatti updated, ready to go.

**Comment by jeffmendoza** [comment]
@jaredbhatti rebased

**Comment by jeffmendoza** [comment]
@steveperry-53 Jared is in meetings all day today, incase you have time.

**Comment by steveperry-53** [comment]
@jeffmendoza, Do you want to squash?

**Comment by jeffmendoza** [comment]
@steveperry-53 Squashed

**Pull request merged**